civihr_default_theme/assets/sass/partials/_panel.scss

```diff
@@ -196,6 +205,10 @@ form {
     }
 }
 
+.chr_panel__header-subtitle {
```

Comment:
The SASS ampersand could be used here as well, inserting this into the parent block ".chr_panel"